numberSign.c

```diff
@@ -0,0 +1,21 @@
+#include <stdio.h>
```

Comment:
Great job! You followed all the guidelines correctly and handled input correctly - once again, just remember to add a comment with your name and function of this program for good practice.